visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html

```diff
@@ -20,7 +20,7 @@
 				class="metric-search"
 				ng-model="$ctrl._viewModel.searchTerm"
 				type="text"
-				placeholder="Metric (highest value)"
+				placeholder="Edge Metric (highest value)"
```

Comment:
I wonder what the `highest value` actually stands for. Might that be a legacy leftover?

Reply:
It is a Kind of legend. In the metrics List you can See the highest value for mcc for instance in brackets.

Reply:
Let‘s Remove it

Reply:
Good point. I just let someone have a look at it and they told me it's better to have the description than removing it.